tests/test_fields.py

```diff
@@ -268,6 +292,13 @@ def test_download(self):
             self.assertEqual(serializer.data['file'], encoded_source)
         finally:
             os.remove('im.jpg')
+                
+    def test_not_found(self):
```

Comment:
What about mocking `django.core.files.base.File.open` here and using `side_effect` we can make it raise a `FileNotFoundError`.